Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'devise'
 gem 'exception_notification'
 gem 'ffi'
 gem 'haml'
+gem 'humanize'
```

Comment:
Instead of adding a dependency, can we use something built into Rails for this? Like `number_to_human`?

Reply:
`number_to_human` does something slightly different: it turns numbers like 12345 into "12.3 Thousand".

Humanize, on the other hand, turns numbers like 25 into "twenty-five". Publications commonly have style requirements that call for writing out numerals less than 100 or less than 10, and I've always thought it's a really nice touch for a website to take enough care to reproduce that.